CHANGELOG.md

```diff
@@ -8,6 +8,9 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+### Changed
+
+- Update apiextensions to v3.30 to introduce `v1alpha3` GS AWS CR's.
```

Comment:
```suggestion
- Update apiextensions to v3.30 to introduce `v1alpha3` GS AWS CR's.
```suggestion
- Update apiextensions to v3.30 to introduce `v1alpha3` GS AWS CR's.

```
```